exercises/practice/bob/.approaches/answer-list/content.md

```diff
@@ -0,0 +1,67 @@
+# Answer list
+
+```python
+_ANSWERS = ['Whatever.', 'Sure.', 'Whoa, chill out!',
+            "Calm down, I know what I'm doing!"]
+
+
+def response(hey_bob):
+    hey_bob = hey_bob.rstrip()
+    if not hey_bob:
+        return 'Fine. Be that way!'
+    isShout = (2 if hey_bob.isupper() else 0)
+    isQuestion = (1 if hey_bob.endswith('?') else 0)
+    return _ANSWERS[isShout + isQuestion]
+
+```
+
+In this approach you define a [list][list] that contains Bob’s answers, and each condition is given a score.
+The correct answer is selected from the list by using the score as the list index.
+
+Python doesn't _enforce_ having real constant values,
+but the `_ANSWERS` list is defined with all uppercase letters, which is the naming convention for a Python [constant][const].
+It indicates that the value is not intended to be changed.
+Python also does not have real [private][private] variables,
```

Comment:
we just shouldn't be using a "private" flag here.  Not unless it's internal to a class or function.